modules/n1ql/pages/n1ql-language-reference/searchfun.adoc

```diff
@@ -142,7 +142,7 @@ For more details, refer to xref:fts:fts-sorting.adoc[Sorting Query Results].
 
 [NOTE]
 ====
-When specifying a complete full text search request with the N1QL SEARCH() function, if the value of the `size` parameter is greater than the xref:fts:fts-response-object-schema.adoc#request[maximum number of full text search results], the query ignores the `size` parameter and returns all matching results.
+When specifying a complete full text search request with the N1QL SEARCH() function, if the value of the `size` parameter is greater than the xref:fts:fts-advanced-settings-bleveMaxResultWindow.adoc[bleveMaxResultWindow], the query ignores the `size` parameter and returns all matching results.
```

Comment:
This is not entirely accurate. 

- If size/from (pagination) settings aren't set, the search index will "stream" all qualifying results to N1QL which can perform further filtering.
- If `size/limit`, `from/offset` are set within the search request within the SEARCH function, FTS handles the pagination.
- N1QL can hand down LIMIT/OFFSET settings to FTS. If this `limit+offset <=  bleveMaxResultWindow` - then the FTS index is deemed pageable and N1QL hands down these settings to FTS. If not, then FTS will stream results to N1QL and N1QL will be responsible for pagination and ordering.

I'll let you fix the content here based on the above.